src/LibraryManager/Configuration/Settings.cs

```diff
@@ -64,24 +64,18 @@ protected Settings()
                 }
                 catch (JsonReaderException)
                 {
-                    // If there were any errors, we'll reset the file
+                    // If there were any errors, we'll take the defaults
+                    // TODO: log a warning
```

Comment:
Definitely would be nice, at least dump it to the JSON output window via ILanguageHost?

Reply:
That only would help if it's installed by VS, but for most users it's either through the CLI or the MSBuild tasks (even building in VS, the build happens out of proc). The only times we install packages within VS are via the wizard (1st time, since it's not reentrant) or via save-on-edit in libman.config.

We need better logging in general, it's nearly impossible to diagnose most of the failures reported without getting a debugger (and a consistent repro) involved.